### PR TITLE
AC-219345: updating files for 2021 R2

### PR DESCRIPTION
**Pull request by PetrKorchagin**

### CODE REVIEW

File: Customization/T280/PhoneRepairShop_Code/.vs/PhoneRepairShop_code/config/applicationhost.config [hunk 1]
```diff
@@ -17,8 +17,10 @@
           %SYSTEMDRIVE% - The drive letter of %IIS_BIN%
 
 -->
+
```
Comment (k-s-popova) [inline Customization/T280/PhoneRepairShop_Code/.vs/PhoneRepairShop_code/config/applicationhost.config]: .vs folder should be excluded from the commit

File: Customization/T280/PhoneRepairShop_Code/PhoneRepairShop_Code.Tests/PhoneRepairShop_Code.Tests.csproj [hunk 2]
```diff
@@ -36,21 +36,24 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="PX.Common">
-      <HintPath>..\..\..\..\..\PhoneRepairShop\Bin\PX.Common.dll</HintPath>
+      <HintPath>..\..\..\..\Bin\PX.Common.dll</HintPath>
```
Comment (k-s-popova) [inline Customization/T280/PhoneRepairShop_Code/PhoneRepairShop_Code.Tests/PhoneRepairShop_Code.Tests.csproj:39]: Why paths to dlls have been changed?

File: Customization/T280/PhoneRepairShop_Code/PhoneRepairShop_Code/DAC/RSSVSetup.cs [hunk 3]
```diff
@@ -1,93 +1,94 @@
...
+
+        #region LastModifiedDateTime
+        [PXDBLastModifiedDateTime()]
+        public virtual DateTime? LastModifiedDateTime { get; set; }
+        public abstract class lastModifiedDateTime : PX.Data.BQL.BqlDateTime.Field<lastModifiedDateTime> { }
+        #endregion
+
+        #region Tstamp
+        [PXDBTimestamp()]
+        [PXUIField(DisplayName = "Tstamp")]
```
Comment (k-s-popova) [inline Customization/T280/PhoneRepairShop_Code/PhoneRepairShop_Code/DAC/RSSVSetup.cs]: No display name should be here.

**Review by k-s-popova — CHANGES_REQUESTED** [review]

File: Customization/T280/PhoneRepairShop_Code/PhoneRepairShop_Code.Tests/PhoneRepairShop_Code.Tests.csproj (hunk 2 above)
Comment (PetrKorchagin) [reply Customization/T280/PhoneRepairShop_Code/PhoneRepairShop_Code.Tests/PhoneRepairShop_Code.Tests.csproj:39]: In fact, they have not changed. The old and new paths point to the same folder.

**Pull request merged**